pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package pir;
+
+message Payload {
+  repeated bytes query = 1;
+}
+
+message SessionPayload {
+  required Payload query = 1;
```

Comment:
Avoid required fields whenever possible. If we ever change the format of the query it'll be impossible to change. Please also add a description of this field.

Reply:
That field is mandatory for the payload. Maybe another message is better if we break the interface.